### PR TITLE
Mo branch codecov

### PR DESCRIPTION
**Pull request by odinlake**
- Add code coverage for pushes to branch.
- Bump version to 1.0.

### CODE REVIEW

**Comment by odinlake** [comment]
After approving and merging this PR you can create a release for tag "v1.0". If the tag has exactly that name (same version as __version__ in python but with a "v" in front) then it should be automatically built and uploaded to PyPi.

**Comment by paul-sheridan** [comment]
Looks all good to me.

**Review by paul-sheridan — COMMENTED** [review]
Looks good to me.

File: .github/workflows/coverage-branch.yaml
Comment (paul-sheridan) [inline .github/workflows/coverage-branch.yaml:1]: Looks good.

**Review by paul-sheridan — APPROVED** [review]
Looks good.

**Pull request merged**